### ISSUE TITLE
Creepers Not Running Away From Felines

### ISSUE THREAD

**Issue by ChillFava** [EdwinMindcraft/origins-architectury#35]
Hey, I was playing with the feline race earlier and noticed that creepers were barely ever running away from them? Not sure if this is a bug or intentional but they still seem to sneak up on you and explode as usual.

**Comment by ChillFava** [comment]
I tried it out some more, it seems like they do run away most of the time but sometimes they just don't? I think it has something to do with them running into a block but instead of climbing over it or going around it they just stay there and then blow up because they're close enough to the player.

**Comment by EdwinMindcraft** [comment]
Yeah, that's just Minecraft's Pathfinding being busted. Them blowing up is technically part of the fabric version, so I won't do anything for it on 1.16.5. Maybe for 1.17, depending on how far I move away from the original code.
